pkg/agent/datamodel/sig_config.go

```diff
@@ -236,6 +236,7 @@ const (
 
 	Windows2019SIGImageVersion string = "17763.3406.220913"
 	Windows2022SIGImageVersion string = "20348.1006.220913"
+	Windows2022Gen2SIGImageVersion string = "20348.1006.220930"
```

Comment:
You may check the version value from the production pipeline run of the Gen 2 VHD.